packages/vuetify/src/mixins/menuable.ts

```diff
@@ -299,7 +325,7 @@ export default Vue.extend({
       return window.pageYOffset ||
         document.documentElement.scrollTop
     },
-    getRoundedBoundedClientRect (el) {
+    getRoundedBoundedClientRect (el: Element): Record<string, number> {
```

Comment:
there should be specified interface for the returned object (top/left/bottom/right/width/height)

Reply:
seems like it maybe `ClientRect` ?

Reply:
Looks like it. Not sure why values are rounded here, but it's not in the scope of this issue